frontend/app/routes/_gcweb-app.personal-information.mailing-address.edit.tsx

```diff
@@ -132,67 +179,101 @@ export default function PersonalInformationMailingAddressEdit() {
   return (
     <>
       {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
-      <Form className="max-w-prose" method="post">
-        <div className="checkbox gc-chckbxrdio">
-          <input id="copy-home-address" type="checkbox" name="ifCopyHomeAddress" checked={isCopyAddressChecked} onChange={checkHandler} />
-          <label id="copy-home-address" htmlFor="copy-home-address">
-            Copy home address
-          </label>
-        </div>
-
-        <InputField
-          id="address"
-          label={t('personal-information:mailing-address.edit.field.address')}
-          name="address"
-          required
-          key={isCopyAddressChecked ? homeAddressInfo?.address : addressInfo?.address}
-          defaultValue={defaultValues.address}
-          errorMessage={errorMessages.address}
-        />
-        <InputField
-          id="city"
-          label={t('personal-information:mailing-address.edit.field.city')}
-          name="city"
-          required
-          key={isCopyAddressChecked ? homeAddressInfo?.city : addressInfo?.city}
-          defaultValue={defaultValues.city}
-          errorMessage={errorMessages.city}
-        />
-        <InputField
-          id="province"
-          label={t('personal-information:mailing-address.edit.field.province')}
-          name="province"
-          key={isCopyAddressChecked ? homeAddressInfo?.province : addressInfo?.province}
-          defaultValue={defaultValues.province}
-          errorMessage={errorMessages.province}
-        />
-        <InputField
-          id="postalCode"
-          label={t('personal-information:mailing-address.edit.field.postal-code')}
-          name="postalCode"
-          key={isCopyAddressChecked ? homeAddressInfo?.postalCode : addressInfo?.postalCode}
-          defaultValue={defaultValues.postalCode}
-          errorMessage={errorMessages.postalCode}
-        />
-        <InputField
-          id="country"
-          label={t('personal-information:mailing-address.edit.field.country')}
-          name="country"
-          required
-          key={isCopyAddressChecked ? homeAddressInfo?.country : addressInfo?.country}
-          defaultValue={defaultValues.country}
-          errorMessage={errorMessages.country}
-        />
-
-        <div className="flex flex-wrap gap-3">
-          <button id="change-button" className="btn btn-primary btn-lg">
-            {t('personal-information:mailing-address.edit.button.change')}
-          </button>
-          <Link id="cancel-button" to="/personal-information" className="btn btn-default btn-lg">
-            {t('personal-information:mailing-address.edit.button.cancel')}
-          </Link>
-        </div>
-      </Form>
+      {isCopyAddressChecked && (
+        <Form className="max-w-prose" method="post">
```

Comment:
Some elements are repeated in the `isCopyAddressChecked` and `!isCopyAddressChecked` blocks that we should leave outside of these blocks, such as the `<Form>` element and the "Change" and "Cancel" buttons.

Reply:
fixed